recipes/velocypack/conda_build_config.yaml

```diff
@@ -0,0 +1,2 @@
+#MACOSX_DEPLOYMENT_TARGET:  # [osx]
+#  - '10.6'                # [osx]
```

Comment:
could you please delete this file (as you are not using it)?

Reply:
done

Reply:
It is still here xD

Reply:
Hopefully now the file is gone. :)